.github/workflows/buildCSharp.yml

```diff
@@ -7,33 +7,50 @@ on:
 jobs:
   build:
     name: "Build and release CSharp SDK"
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
 
     steps:
       - name: "Checkout develop Branch"
         uses: actions/checkout@v2
         with:
           ref: develop
-        
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.x'
+
       - name: Setup CSharp
+        shell: bash
         run: |
-          wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+          set -x
+          cmake -E make_directory ./build
+          wget https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
           sudo dpkg -i packages-microsoft-prod.deb
-          sudo apt-get update; \
-          sudo apt-get install -y apt-transport-https && \
-          sudo apt-get update && \
+          sudo apt-get update
+          sudo apt-get install -y apt-transport-https
           sudo apt-get install -y dotnet-sdk-3.1
-
+          
```

Comment:
I deleted a second call to apt update, but I haven't really tested that it's not needed.

Also, this might be enough to do `sudo apt-get install -y apt-transport-https dotnet-sdk-3.1`

Reply:
I got this part from dotnet documentation, didn't really think about it while just using it 🤔 .
https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu